pkg/apis/stork/v1alpha1/migration.go

```diff
@@ -23,6 +23,7 @@ type MigrationSpec struct {
 	SkipServiceUpdate            *bool             `json:"skipServiceUpdate"`
 	IncludeNetworkPolicyWithCIDR *bool             `json:"includeNetworkPolicyWithCIDR"`
 	Selectors                    map[string]string `json:"selectors"`
+	ExcludeResources             map[string]string `json:"excludeResources"`
```

Comment:
How about `ExcludeSelectors` ? The behavior is very similar to the existing `Selectors` flag, except it is reverse.